themes/default/Post.template.php

```diff
@@ -476,6 +476,7 @@ function template_add_new_attachments()
 		numOfAttachmentAllowed: ' . $context['attachments']['num_allowed'] . ',
 		totalAttachSizeUploaded: ' . (isset($context['attachments']['total_size']) && !empty($context['attachments']['total_size']) ? $context['attachments']['total_size'] : 0) . ',
 		numAttachUploaded: ' . (isset($context['attachments']['quantity']) && !empty($context['attachments']['quantity']) ? $context['attachments']['quantity'] : 0) . ',
+		fileDisplayTemplate: \'<div class="statusbar"><div class="info"></div><div class="progressBar"><div></div></div><div class="control fa fa-times-circle"></div></div>\',
```

Comment:
Very well I guess.
This was just a copy&paste from the js file of code written a while ago.
Any hint since I don't remember the code to use? lol